.travis.yml

```diff
@@ -9,10 +9,12 @@ install:
   - $(go env GOPATH)/bin/golangci-lint --version
 script:
   - $(go env GOPATH)/bin/golangci-lint run --disable wsl ./...
+  - go clean
   - go build ./...
-  - go test ./... --short
+  - go test ./... --short -count=1
```

Comment:
What does this change actually do? The explanation of this must show up in the squashed message after the merge so we know what the purpose here is.

Reply:
It's disabling the caching of test results